Jenkinsfile.cloud

```diff
@@ -166,6 +166,8 @@ node(NODE) {
     parallel par_stages; par_stages = [:]
 
     stage("Finalizing, generate metadata") {
+        // See https://github.com/openshift/installer/blob/master/Documentation/dev/libvirt-howto.md#13a-rhcos
+        sh "qemu-img resize ${img_prefix}-qemu.qcow2 +8G"
```

Comment:
The gzip command below is still taking in `${qcow}` which is `${img_prefix}.qcow2`